### PR TITLE
Website stripe plot

### PR DESCRIPTION
**Pull request by ceresek**
Sorry, wrong use of revert and merge, should be OK now.

### CODE REVIEW

File: website/_layouts/mainrenaissance.html [hunk 1]
```diff
@@ -141,7 +141,13 @@
   <br/>
   <br/>
 
-  <!-- summary of latest benchmark results -->
+  <div id="measurements" class="text-snippet">
+    <img src="https://github.com/renaissance-benchmarks/measurements/raw/master/stripe.png"/>
```
Comment (axel22) [inline website/_layouts/mainrenaissance.html]: One detail - with the height 500px, the width will be ~1667px, and our current CSS tiling framework is 960px wide.

If we reduced the width to at least 1100px, the figure would spill only a little, while still being readable.

Comment (axel22) [reply website/_layouts/mainrenaissance.html]: Example (by editing the Github page directly in the browser):

screenshot-21-54-10

Comment (ceresek) [reply website/_layouts/mainrenaissance.html]: Tried on a local copy but width 1100px does not look very readable in my browser (see screenshot), is it possible we have different zoom settings ?

screenshot

Comment (ceresek) [reply website/_layouts/mainrenaissance.html]: If we do not want to exceed the 960px I think we should use two rows.

**Comment by axel22** [comment]
Two rows are probably the right solution.

**Comment by ceresek** [comment]
Will do.

**Pull request merged**